doc/input-output-reference/src/overview/group-plant-equipment.tex

```diff
@@ -3027,7 +3027,7 @@ \subsubsection{Inputs}\label{inputs-8-016}
 
 \paragraph{Field: Condenser Outlet Node Name}\label{field-condenser-outlet-node-name-7}
 
-This required alpha field contains the identifying name for the chiller condenser side outlet node.
+This required alpha field contains the identifying name for the chiller condenser side outlet node. If the chiller is AirCooled, a node name must be specified but is not used.
```

Comment:
@JasonGlazer Why not change the getinput to allow this field to be blank for aircooled?